File: src/easy/1758-minimum-changes-to-make-alternating/python/main.py
class Solution:
    def minOperations(self, s: str) -> int:
        a = 0
        b = 0
        for i in range(len(s)):
            if s[i] != str(i % 2):
                a += 1
            else:
                b += 1
        return min(a, b)

File: src/easy/1758-minimum-changes-to-make-alternating/typescript/main.ts
function minOperations(s: string): number {
	let a = 0, b = 0; // eslint-disable-line one-var, one-var-declaration-per-line
	for (const [i, c] of s.split('').entries()) {
		if (c === (i % 2).toString()) {
			a++;
		} else {
			b++;
		}
	}

	return Math.min(a, b);
}
